package.json

```diff
@@ -18,6 +18,11 @@
     "version": "auto-changelog -p --tag-prefix='' && git add CHANGELOG.md",
     "precommit:tartufo": "node ./bin/tartufo-shim.js pre-commit"
   },
+  "files": [
+    "bin/*",
+    "lib/*",
+    "SECURITY.md"
+  ],
```

Comment:
package.json, the file(s) specified in `main`, `README.md`, `LICENSE`, and `CHANGELOG.md` are _always_ included regardless of any configuration.